Lib/test/test_logging.py

```diff
@@ -4487,6 +4490,15 @@ def test_multiprocessing(self):
             if multiprocessing_imported:
                 import multiprocessing
 
+        if first_pass:  # https://bugs.python.org/issue45128
+            import multiprocessing.queues
```

Comment:
Importing multiprocessing.queues can fail and has a side effect: it imports multiprocessing.queues which may not be already imported. As I wrote previously, I would prefer to remove these tests. I don't think that we need regression tests on the test suite itself. Just fix the issue, that's it.

Reply:
Sure! Done!